src/V1/Actor/Template/Builder.php

```diff
@@ -54,16 +56,37 @@ private function buildActorTemplateFilePathCandidates(): array
         $actorTemplateFilePathCandidates = [];
 
         $templateTreeMap = $this->getTemplateTreeMapRepository()->get();
-        foreach($templateTreeMap as $templateTree) {
-            $actorTemplateFilePathCandidate = sprintf(
-                '%s/%s',
-                $templateTree->getDirectoryPath(),
-                $this->getFabricationFileActor()->getTemplateRelativeFilePath()
-            );
 
-            $actorTemplateFilePathCandidates[] = $actorTemplateFilePathCandidate;
+        // Let the Fabrication File preferences be used over global ones
```

Comment:
Method: "If the Fabrication File specifies a list of Preferred Template Tree directory paths by using their associated names I collect those in an array in order first, using their paths as keys and concatenating the Template's relative file path to them by asking buildTemplatePath to do that for me, then I add those specified globally to the same array, in the same manner. The way PHP arrays work will deduplicate the globally specified paths, preserving global ordering and existence when not usurped by a Fabrication File preference. If a Preferred Template Tree does not specify a name that exists in the global specification, I throw an exception. I return an array of possible Template file paths. I assume another Actor will validate if they exist on the FS."